docker-compose.yml

```diff
@@ -41,75 +42,96 @@ services:
+  #     - ./redis-insight/data:/var/lib/redis-insight
+  #   networks:
+  #     - gobel_link
+  #   restart: always
+  #   depends_on:
+  #     - logstash
   nginx:
     container_name: "${GOBEL_NGINX_CONTAINER_NAME}"
+    environment: 
+      - GOBEL_NGINX_HOST_PORT=${GOBEL_NGINX_HOST_PORT}
+      - GOBEL_NGINX_ADMIN_CLIENT_SERVER_NAME=${GOBEL_NGINX_ADMIN_CLIENT_SERVER_NAME}
+      - GOBEL_NGINX_ADMIN_CLIENT_ACCESS_LOG_NAME=${GOBEL_NGINX_ADMIN_CLIENT_ACCESS_LOG_NAME}
+      - GOBEL_NGINX_ADMIN_CLIENT_ERROR_LOG_NAME=${GOBEL_NGINX_ADMIN_CLIENT_ERROR_LOG_NAME}
+      - GOBEL_NGINX_API_SERVER_NAME=${GOBEL_NGINX_API_SERVER_NAME}
+      - GOBEL_NGINX_API_ACCESS_LOG_NAME=${GOBEL_NGINX_API_ACCESS_LOG_NAME}
+      - GOBEL_NGINX_API_ERROR_LOG_NAME=${GOBEL_NGINX_API_ERROR_LOG_NAME}
+      - GOBEL_API_CONTAINER_NAME=${GOBEL_API_CONTAINER_NAME}
+      - GOBEL_API_CONTAINER_PORT=${GOBEL_API_CONTAINER_PORT}
+      - GOBEL_NGINX_CLIENT_SERVER_NAME=${GOBEL_NGINX_CLIENT_SERVER_NAME}
+      - GOBEL_NGINX_CLIENT_ACCESS_LOG_NAME=${GOBEL_NGINX_CLIENT_ACCESS_LOG_NAME}
+      - GOBEL_NGINX_CLIENT_ERROR_LOG_NAME=${GOBEL_NGINX_CLIENT_ERROR_LOG_NAME}
+      - GOBEL_CLIENT_CONTAINER_NAME=${GOBEL_CLIENT_CONTAINER_NAME}
+      - GOBEL_CLIENT_CONTAINER_PORT=${GOBEL_CLIENT_CONTAINER_PORT}
     build:
         context: "./nginx"
         dockerfile: "Dockerfile"
         args:
+          VUE_APP_API_ENDPOINT: "${GOBEL_API_HTTP_API_SCHEME}://${GOBEL_NGINX_API_SERVER_NAME}"
           GOBEL_NGINX_IMAGE_NAME: "${GOBEL_NGINX_IMAGE_NAME}"
           GOBEL_NGINX_IMAGE_TAG: "${GOBEL_NGINX_IMAGE_TAG}"
-          VUE_APP_API_ENDPOINT: "${GOBEL_NGINX_VUE_APP_API_ENDPOINT}"
           GOBEL_ADMIN_CLIENT_IMAGE_NAME: "${GOBEL_ADMIN_CLIENT_IMAGE_NAME}"
           GOBEL_ADMIN_CLIENT_IMAGE_TAG: "${GOBEL_ADMIN_CLIENT_IMAGE_TAG}"
           GOBEL_ADMIN_CLIENT_PATH_TO_DIST: "${GOBEL_ADMIN_CLIENT_PATH_TO_DIST}"
-          GOBEL_NGINX_IMAGE_NAME: "${GOBEL_NGINX_IMAGE_NAME}"
-          GOBEL_NGINX_IMAGE_TAG: "${GOBEL_NGINX_IMAGE_TAG}"
           GOBEL_NGINX_ADMIN_CLIENT_ACCESS_LOG_NAME: ${GOBEL_NGINX_ADMIN_CLIENT_ACCESS_LOG_NAME}
           GOBEL_NGINX_ADMIN_CLIENT_ERROR_LOG_NAME: ${GOBEL_NGINX_ADMIN_CLIENT_ERROR_LOG_NAME}
           GOBEL_NGINX_API_ACCESS_LOG_NAME: ${GOBEL_NGINX_API_ACCESS_LOG_NAME}
           GOBEL_NGINX_API_ERROR_LOG_NAME: ${GOBEL_NGINX_API_ERROR_LOG_NAME}
           GOBEL_NGINX_CLIENT_ACCESS_LOG_NAME: ${GOBEL_NGINX_CLIENT_ACCESS_LOG_NAME}
           GOBEL_NGINX_CLIENT_ERROR_LOG_NAME: ${GOBEL_NGINX_CLIENT_ERROR_LOG_NAME}
+    volumes:
+      - ./nginx/template:/etc/nginx/templates
     ports:
       - "${GOBEL_NGINX_HOST_PORT}:${GOBEL_NGINX_CONTAINER_PORT}"
     networks:
       - gobel_link
     restart: always
     logging:
-      driver: "fluentd"
+      driver: "gelf"
       options:
-        fluentd-address: ${GOBEL_FLUENTD_ADDRESS}
-        fluentd-async-connect: "true"
-        tag: "gobel.nginx"
+        gelf-address: udp://localhost:${GOBEL_LOGSTASH_HOST_PORT}
```

Comment:
[memo]
```sh
udp://localhost:${GOBEL_LOGSTASH_HOST_PORT}
// OK localhost NO gobel-logstash(container name)
```

It seems that it needs to be an endpoint that can communicate from the host, not from the container.